javaslang/src/test/java/javaslang/control/EitherRightProjectionTest.java

```diff
@@ -66,8 +67,22 @@ public void shouldGetOnRightProjectionOfRight() {
 
     @Test
     public void shouldRightProjectionOrElseRightProjection() {
-        assertThat(Either.right(1).right().orElse(Either.right(2).right()).get()).isEqualTo(1);
-        assertThat(Either.<Integer, Integer> left(1).right().orElse(Either.<Integer, Integer> right(2).right()).get()).isEqualTo(2);
+        final RightProjection<Object, Integer> rightRight = Either.right(1).right();
```

Comment:
I think you could inline these two.
Also, `Either.right(1)` could more simply be written as `Right(1)` (with static import to `API`)